study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/EventBuilder.java

```diff
@@ -125,16 +125,20 @@ private long insertEventTrigger(Handle handle, Config triggerCfg) {
     private long insertEventAction(Handle handle, Config actionCfg) {
         EventActionDao actionDao = handle.attach(EventActionDao.class);
         String type = actionCfg.getString("type");
+        boolean isDynamicTemplate = false;
+        if (actionCfg.hasPath("is_dynamic")) {
+            isDynamicTemplate = actionCfg.getBoolean("is_dynamic");
```

Comment:
Nice! I like how this is backwards-compatible. Let's also update the `study-example.conf` file as well to document this.